core/templates/base-components/base-content.component.html

```diff
@@ -102,3 +102,8 @@
     Dev Mode
   </div>
 </main>
+<style>
+  .disflx {
```

Comment:
disflx is not really a good name. Can you try to come up with a better one? Maybe flex-display?

Reply:
i have changed it to flex-display